lib/scrum_board.rb

```diff
@@ -12,6 +12,10 @@ def columns
     @columns ||= @board_data['lists'].map{|x| Column.new(@board_data, x['id'], @settings)}
   end
 
+  def sprint_backlog_column
+    columns.select{ |col| col.name == 'Sprint Backlog' }.first
```

Comment:
I would prefer to write `column` instead of `col`, more explicit 😉 

The column name can be configured, as you can see in the [Documentation](https://github.com/openSUSE/trollolo#default-configuration), so you should use `settings.list_names['planning_backlog']` 😄

Reply:
Hello @Ana06, the method list_names does not exist in the setting object, what I did was `@settings.scrum['list_names']['planning_backlog']`